src/server/utilities/canAdvertise/index.js

```diff
@@ -0,0 +1,15 @@
+const canAdvertise = (headers, queryParam) => {
+  if (queryParam && queryParam['force-ads'] === 'true') {
```

Comment:
This should be removed - we cannot check a query param unless mozart passes this through and adds it to the routing hash.